libsolidity/interface/Exceptions.cpp

```diff
@@ -33,22 +33,22 @@ Error::Error(Type _type, SourceLocation const& _location, string const& _descrip
 	switch(m_type)
 	{
 	case Type::DeclarationError:
-		m_typeName = "Declaration Error";
```

Comment:
I am fine adding another method which returns it without spaces. That is in the required by the standard json spec.